visualization/visualization_test.go

```diff
@@ -97,6 +97,42 @@ func TestGenerateFlameGraphExecError(t *testing.T) {
 	mockExecutor.AssertExpectations(t)
 }
 
+func TestRunPerlScriptDoesExist(t *testing.T) {
+	mockOSWrapper := new(mockOSWrapper)
+	executor := defaultExecutor{
+		osWrapper: mockOSWrapper,
+	}
+	cwd, _ := os.Getwd()
```

Comment:
sure, i just mean it's bad form to discard errors, and you would want to fail fast here to indicate that something is messed up with your test environment